src/blocks/food-and-drinks/food-item/icons.js

```diff
@@ -10,9 +10,7 @@ const icons = {};
 
 icons.glutenFree = (
 	<SVG
-		height="14"
```

Comment:
Does removing these height/widths cause deprecation issues?

Reply:
We should test to confirm; I have a feeling it may.

Reply:
Shoot, you are correct. Deprecation's needed all around.